packages/graphql/src/translate/cypher-builder/references/Param.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CypherContext } from "../CypherContext";
+import { CypherParam } from "./Reference";
+
+export class Param<T = any> implements CypherParam {
```

Comment:
**suggestion (non-blocking):** interesting, I thought this would actually be the other way around - it reads as if a Cypher param should have the behaviours of a param, not the other way around?

```suggestion
export class CypherParam<T = any> implements Param {
```